libsmtutil/SMTLib2Interface.cpp

```diff
@@ -139,7 +139,7 @@ string SMTLib2Interface::toSExpr(Expression const& _expr)
 	std::string sexpr = "(";
 	if (_expr.name == "int2bv")
 	{
-		size_t size = std::stoi(_expr.arguments[1].name);
+		size_t size = std::stoul(_expr.arguments[1].name);
```

Comment:
@leonardoalt why is this used here? To check it is a number?

Reply:
@axic Yes, we need to guarantee that the argument there is a number, because that's what the solver takes.

Reply:
Why not use something more sophisticated? I think these functions can depend on the locale settings too.

Reply:
It some other code we have used `boost::lexical_cast<int>()` to do the parsing.

Reply:
The locale seems to be a superset, and this is code generated by us and not the user, so I would say it's fine as is (plus we get to not use boost ;)). But not completely opposed if consensus to change